collated/main/a0153617e.md

```diff
@@ -0,0 +1,50 @@
+# a0153617e
+###### \java\jym\manager\logic\commands\CompleteCommand.java
+``` java
+public class CompleteCommand extends Command {
+
+    public static final String COMMAND_WORD = "complete";
```

Comment:
Is `public` the best access modifier for `COMMAND_WORD` ? As a rule of thumb start all data members as private, and depending on their usage promote them to protected, and if absolutely necessary, to public.